src/parser/parse_tree_cl.c

```diff
@@ -68,6 +68,8 @@
 #define MAX_STRING_SEGMENT_LENGTH 254
 #define DONT_PRT_LONG_STRING_LENGTH 256
 
+#define PT_APPLY_CHECK_ACTION(parser, ptr, arg)  if((ptr)) (ptr) = pt_walk_private ((parser), (ptr), (arg))
```

Comment:
I think, PT_APPLY_CHECK_ACTION, macro name is hard to understand. How about PT_APPLY_WALK?

Reply:
For multi-line macro function, how about `do { } while (0)`?

Reply:
@byungwook-kim I'll take a good suggestion and fix it.

@hgryoo I had the same thought. However, since it is a simple syntax, it is not difficult to understand, and I did not want to wrap it in an empty loop for this. However, because of byungwook-kim's suggestion, I will edit it anyway, so I will edit it at this time.